packages/website/src/pages/docs/Changelog/v0.8.6.mdx

```diff
@@ -0,0 +1,42 @@
+
+* Added `title` to all [plots](https://www.squiggle-language.com/docs/Api/Plot), and to scales for labeling x and y axes. Added validation for `tickFormat`. ([#2337](https://github.com/quantified-uncertainty/squiggle/pull/2337))
+
+* Improved [mixture()](https://www.squiggle-language.com/docs/Api/Dist#mixture) typing, but limited to 5 parameters maximum ([#2334](https://github.com/quantified-uncertainty/squiggle/pull/2334))
+
+* Added [Dict.pick](https://www.squiggle-language.com/docs/Api/Dictionary#pick), [Dict.omit](https://www.squiggle-language.com/docs/Api/Dictionary#omit) ([#2376](https://github.com/quantified-uncertainty/squiggle/pull/2376))
+
+* [sum](https://www.squiggle-language.com/docs/Api/Number#sum), [product](https://www.squiggle-language.com/docs/Api/Number#product), [sort](https://www.squiggle-language.com/docs/Api/Number#sort), [cumsum](https://www.squiggle-language.com/docs/Api/Number#cumulative-sum), [cumprod](https://www.squiggle-language.com/docs/Api/Number#cumulative-product) all accept lists with 0 elements ([#2351](https://github.com/quantified-uncertainty/squiggle/pull/2351))
+
+* Added [Dist.sum](https://www.squiggle-language.com/docs/Api/Dist#sum), [Dist.product](https://www.squiggle-language.com/docs/Api/Dist#product), [Dist.cumsum](https://www.squiggle-language.com/docs/Api/Dist#cumulative-sum), [Dist.cumprod](https://www.squiggle-language.com/docs/Api/Dist#cumulative-product), [Dist.diff](https://www.squiggle-language.com/docs/Api/Dist#diff) ([#2356](https://github.com/quantified-uncertainty/squiggle/pull/2356))
+
+* Added [List.slice](https://www.squiggle-language.com/docs/Api/List#slice), [Danger.combinations](https://www.squiggle-language.com/docs/Api/Danger#combinations), [Danger.allCombinations](https://www.squiggle-language.com/docs/Api/Danger#allcombinations) ([#2347](https://github.com/quantified-uncertainty/squiggle/pull/2347))
+
+### [@quri/squiggle-components](https://www.npmjs.com/package/@quri/squiggle-components)
+
+* Added calculator class to squiggle-lang and calculator component to squiggle-components. Useful for presenting functions to end-users. ([#2265](https://github.com/quantified-uncertainty/squiggle/pull/2265))
+
+* Breaking: Changed Calculator fields attribute, to inputs, which now requires Input objects. These allow for multiple types of inputs ([#2343](https://github.com/quantified-uncertainty/squiggle/pull/2343))
+
+* Added sampleCount field to Calculator.make ([#2474](https://github.com/quantified-uncertainty/squiggle/pull/2474))
+
+* Added title to all plots, and to scales for xAxisLabel and yAxisLabel. Added validation for tickFormat. ([#2337](https://github.com/quantified-uncertainty/squiggle/pull/2337))
+
+* ([#2300](https://github.com/quantified-uncertainty/squiggle/pull/2300))
```

Comment:
I'd remove this line
https://github.com/quantified-uncertainty/squiggle/pull/2300

Reply:
Uh, right, I patched it in CHANGELOG.md (https://github.com/quantified-uncertainty/squiggle/commit/e10b4ac7da889131b9baba364f0095f983130716) but used the earlier revision for .mdx generator script...

Reply:
Patched on main: b20b9bb93242dcb300991b9a175c33765a60c1b6